docs/quickstart.md

```diff
@@ -13,7 +13,7 @@ starting development using TCMalloc at least run through this quick tutorial.
 
 Running the code within this tutorial requires:
 
-*   A compatible platform (E.g. Linux). Consult the [Platforms Guide](platforms)
+*   A compatible platform (E.g. Linux). Consult the [Platforms Guide](https://github.com/google/tcmalloc/blob/master/docs/platforms.md)
```

Comment:
Based on #3, using just "platforms.md" means that the internal link will work for both the Github source tree view and the Github pages view.